app/src/main/res/values/styles.xml

```diff
@@ -1,10 +1,112 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="ShapeAppearanceOverlay.MyApp.Button.RoundedGreen" parent="">
-        <item name="android:background">@drawable/round_corners</item>
+    <!--    BASE STYLES -->
+
+    <style name="TextNormal" parent="">
```

Comment:
add an indication to the text size to avoid future conflicts when adding different text sizes. i.e: TextMedium